src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -59,7 +63,30 @@ public CommandBase drive(DoubleSupplier forward, DoubleSupplier turn) {
             driveTrain.arcadeDrive(yAxis, xAxis);
         }, this);
     }
-}
 
+    public CommandBase driveDistance(double distance, double speed) {
+        return new FunctionalCommand(() -> {
+            leftMotorGroup.reset();
+            rightMotorGroup.reset();
+        }, () -> {
+            driveTrain.arcadeDrive(speed, 0);
+        }, (interrupted) -> {
+            driveTrain.stopMotor();
+        }, () -> {
+            double avg = (leftMotorGroup.getDistance() + rightMotorGroup.getDistance()) / 2;
+            return (avg >= distance);
+        }, this);
+    }
 
-    
\ No newline at end of file
+    public CommandBase turnDegrees(double degrees, double speed) {
+        return new FunctionalCommand(() -> {
+            gyro.reset();
+        }, () -> {
+            driveTrain.arcadeDrive(0, speed);
+        }, (interrupted) -> {
+            driveTrain.stopMotor();
+        }, () -> {
+            return gyro.getAngle() >= degrees;
```

Comment:
How will this behave if `degrees` is negative?

Reply:
Finished